ansible/roles/ansible-tvault-contego-extension/tasks/object_store_service.yml

```diff
@@ -33,6 +33,25 @@
       when: ansible_distribution == ubuntu and (ansible_lsb.codename != xenial and ansible_lsb.codename != willy and ansible_lsb.codename != bionic)
   when: (object_store == True)
 
+- name: enable and daemon reload tvault contego service
+  block:
+    - shell: systemctl daemon-reload
+    - name: enable tvault-contego service and reload it
+      service: name=tvault-contego  state=restarted enabled=yes
+  when: >
+       (ansible_distribution_major_version=="7" and ansible_distribution== centos) or
+       (ansible_distribution_major_version=="7" and ansible_distribution=="RedHat") or
+       (ansible_distribution== suse) or (ansible_distribution== SLES) or
+       (ansible_distribution=="Ubuntu" and (ansible_lsb.codename == xenial or ansible_lsb.codename == willy or ansible_lsb.codename == bionic))
+
+- name: restart tvault-contego
+  service: name=tvault-contego state=restarted
+  when: >
```

Comment:
service is getting restarted in the above block already, no need to restart it again.

Reply:
We can remove double restart , there is no impact to restart service again